addons/binding/pom.xml

```diff
@@ -33,6 +33,7 @@
     <module>org.openhab.binding.bluetooth.bluegiga.test</module>
     <module>org.openhab.binding.bluetooth.bluez</module>
     <module>org.openhab.binding.bluetooth.blukii</module>
+    <module>org.openhab.binding.bluetooth.ruuvitag</module>
```

Comment:
To have the binding being picked up by the distro, you furthermore need to add it to the feature.xml, again at the alphabetically correct position. If you have a dependency on some transport bundle (e.g. upnp, mdns or serial), make sure to add a line for this dependency as well (see the other bindings as an example).

Reply:
Ah. I completely missed that one. It should be part of the `openhab-binding-bluetooth` feature just like the other bluetooth bindings.

Reply:
Thank you. Added feature in 9945819.